### PR TITLE
feat(demo): Add Feature Store demo

### PR DESCRIPTION
**Pull request by xxhZs**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

add feature store demo

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)
## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: integration_tests/mfa-change-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+If we intend to modify our business logic, we simply need to update the materialized view within our RisingWave by using SQL statements.
+
+## Installation
+
+Run it in local.
+
+1. Start kafka and risingwave. Then create source and mv in risingwave. can run this script
+
+```./run_local.sh```
```
Comment (hzxa21) [inline integration_tests/mfa-change-demo/README.md:19]: I saw that a docker compose file is provided but there is no instruction on how to use it. I think we don't need to provide a way to run the demo without docker. Let's update the README for running the demo with docker.

File: integration_tests/mfa-change-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Build docker. Kafka RisingWave and Feature Store.
+
+```docker compose up --build```
+
+2. Then we can get the simulation results for Feature store in `.log`.
+
+```cat .log/simulator_log```
+
+3. We also can run MFA demo
```
Comment (hzxa21) [inline integration_tests/mfa-change-demo/README.md:25]: Can we explain what MFA is and what the demo does?

File: integration_tests/mfa-change-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+If we intend to modify our business logic, we simply need to update the materialized view within our RisingWave by using SQL statements.
+
+## Installation
+
+Run it in local.
+
+1. Build docker. Kafka RisingWave and Feature Store.
+
+```docker compose up --build```
```
Comment (hzxa21) [inline integration_tests/mfa-change-demo/README.md:19]: Can we explain what the demo is? I know it is mentioned in the blog already but I think it is better to make the demo self-explained

File: integration_tests/mfa-change-demo/README.md (hunk 2 above)
Comment (hzxa21) [reply integration_tests/mfa-change-demo/README.md:25]: We can have a section at the beginning of this README to mention that there are two demos available and explain what they are.

**Review by hzxa21 — APPROVED** [review]
Chatted offline. We should rename the demo folder from `mfa-change-demo` to `feature-store-demo`.

Rest LGTM

File: Cargo.lock [hunk 4]
```diff
@@ -49,9 +49,9 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "1.0.4"
+version = "1.0.5"
```
Comment (BugenZhao) [inline Cargo.lock:52]: Seems you have accidentally bumped all dependencies in this PR. Would you please revert these changes?

**Review by hzxa21 — APPROVED** [review]

**Review by BugenZhao — APPROVED** [review]

**Comment by yuhao-su** [comment]
This pr caused 17 security alterts. Please help resolve them. https://github.com/risingwavelabs/risingwave/security/dependabot
